Helper/FeatureSwitch/Decider.php

```diff
@@ -257,4 +257,8 @@ public function isSaveCartInSections() {
     public function ifShouldDisablePrefillAddressForLoggedInCustomer() {
         return $this->isSwitchEnabled(Definitions::M2_IF_SHOULD_DISABLE_PREFILL_ADDRESS_FROM_BOLT_FOR_LOGGED_IN_CUSTOMER);
     }
+
+    public function isHandleVirtualProductsAsPhysical() {
```

Comment:
I think the function signature should match the constant. Either `handleVirtualProductsAsPhysical` or change the constant to `M2_IS_VIRTUAL_PRODUCTS_HANDLED_AS_PHYSICAL` (reworded to sound a bit better).

I think I prefer the former but it's up to you.

Reply:
Done